### PR TITLE
Feature/statement of case/adultery

### PR DESCRIPTION
**Pull request by lukaszdrynski**

### CODE REVIEW

File: app/views/v1/exit/adultery-reason-time.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "exit.html" %}
+
+{% block heading %}
+You cannot use adultery as a reason for divorce
+{% endblock %}
+
+{% block exit_reason %}
+<p class="text">
+    You can't use adultery as a reason for divorce if you continued to live together as a couple with your husband/wife
```
Comment (sophiedrake) [inline app/views/v1/exit/adultery-reason-time.html]: `{{ husbandOrWife() }}`

File: app/views/v1/statement-of-case/adultery/address/postcode.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+{% block form %}  
+
+  <p>The court will send the documentation to this address</p>
+  <fieldset class="form-group">
+    <label class="form-label" for="postcode">Postcode</label>
+    <input class="form-control"
+           id="postcode"
+           name="postcode"
+           type="text"
+           value="{{ session.petitioner.postcode }}">
```
Comment (sophiedrake) [inline app/views/v1/statement-of-case/adultery/address/postcode.html]: `session.otherparty` or something similar?

File: app/views/v1/statement-of-case/adultery/supporting-information.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    <input class="form-control" id="where-occurred" name="whereOccurred"
+           type="text" value="{{ session.statementOfCase.whereOccurred }}">
+</fieldset>
+
+<fieldset class="form-group">
+    <label class="form-label-bold"
+           for="why-left">
+        Describe what happened
+    </label>
+    <label class="form-hint">
```
Comment (sophiedrake) [inline app/views/v1/statement-of-case/adultery/supporting-information.html]: `class="form-hint text"`

**Comment by anperez78** [comment]
👍

**Pull request merged**